CMakeLists.txt

```
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(movegroup_interface_demo)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  geometry_msgs
+  roscpp
+  rospy
+  std_msgs
+  moveit_ros_planning_interface
+)
+
+
+###################################
+## catkin specific configuration ##
+###################################
+## The catkin_package macro generates cmake config files for your package
+## Declare things to be passed to dependent projects
+## INCLUDE_DIRS: uncomment this if your package contains header files
+## LIBRARIES: libraries you create in this project that dependent projects also need
+## CATKIN_DEPENDS: catkin_packages dependent projects also need
+## DEPENDS: system dependencies of this project that dependent projects also need
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES movegroup_interface_tutorial
+#  CATKIN_DEPENDS geometry_msgs roscpp rospy std_msgs
```

Need to declare catkin dependencies for other packages which may include this one. It seems that only geometry_msgs are used in this package.
```suggestion
  CATKIN_DEPENDS geometry_msgs roscpp
```